BoxContentSDK/BoxContentSDK/Requests/BOXCollaborationRemoveRequest.m

```diff
@@ -8,12 +8,6 @@
 #import "BOXAPIJSONOperation.h"
 #import "BOXContentSDKConstants.h"
 
-@interface BOXCollaborationRemoveRequest ()
-
-@property (nonatomic, readonly, strong) NSString *collaborationID;
```

Comment:
I think the convention we ended up settling on is readonly in .h, and readwrite in .m

If people really hate it we can go clean it up later, but for now let's do that for consistency.